.expeditor/scripts/chef-infra-client.spec

```diff
@@ -0,0 +1,89 @@
+# Disable any shell actions, replace them with simply 'true'
+%define __spec_prep_post true
+%define __spec_prep_pre true
+%define __spec_build_post true
+%define __spec_build_pre true
+%define __spec_install_post true
+%define __spec_install_pre true
+%define __spec_clean_post true
+%define __spec_clean_pre true
+
+# Use SHA256 checksums for all files
+%define _binary_filedigest_algorithm 8
+
+%define _binary_payload w1.xzdio
+
+# Disable creation of build-id links
+%define _build_id_links none
+
+Name:           chef-infra-client
+Version:        %{VERSION}
+Release:        1%{?dist}
+Summary:        The full stack of Chef Infra Client
+AutoReqProv: 	no
+BuildRoot: 	    %buildroot
+Prefix: 	    /
+Group: 		    default
+License:        Chef EULA
+URL: 		    https://www.chef.io
+Packager: 	    Chef Software, Inc. <maintainers@chef.io>
+Source0:        %{CHEF_INFRA_TAR}
+Source1:        %{CHEF_MIGRATE_TAR}
+
+%description
+The full stack of Chef Infra Client
```

Comment:
Probably need to check with Product on the wording of the description and summary, I'm not sure that 'full stack of ...' are the right words anymore.

Reply:
I've updated the summary and description to:
"Chef Infra Client is an agent that runs locally on each node managed by the Chef Infra Server."

I will make further updates based on product input later.